tools/ci/before_script.sh

```diff
@@ -1,7 +1,11 @@
 set -v
 
 if [[ -n "$TEST_SUITE" ]]; then
-  bundle exec rake test:$TEST_SUITE:setup
+  if [[ -n "$PARALLEL" ]]; then
+    bundle exec parallel_test -e "bundle exec rake test:$TEST_SUITE:setup"
```

Comment:
TODO: This is cumbersome to run locally, find a better way so developers can run this locally.